### ISSUE TITLE
snowpack spinup based on the 26 series runs

### ISSUE THREAD

**Issue by adamrher** [NCAR/amwg_dev#188]
### Description of the run

Should we generate some CLM initial conditions with spun-up snowpack? This is a two-step process:

1. Run 20 year F or B case with hi-frequency coupler output for driving an I compset
2. Run CLM standalone, cycling over streams from the forcing run until the snowpack is spun-up (~500 years)

I am going to propose a B case 20 year forcing run, similar to 26d. That is, I am going to reset_snow / reset_snow_glc for the forcing run. The reason I advocate for resetting the snow in the forcing run, is I want to (if anything) bias the snow pack towards the warm side (favoring less snowpack and freshwater runoff to the LabSea). But I could be talked out of this.

In (1), we need to set namelist to setup the hi-freq coupled history for driving CLM standalone. I've done this with MCT in the past by setting these in user_nl_cpl
```
histaux_a2x3hr=.true.
histaux_a2x24hr=.true.
histaux_a2x1hr=.true.
histaux_a2x1hri=.true.

```
I am not sure whether this how you generate the forcing streams with NUOPC though. @mvertens @cecilehannay @JulioTBacmeister @gustavo-marques @fobryan3 @klindsay28 @billsacks @ekluzek @olyson 

### Suffix in the casename

Suffix

### Namelist modifications

Namelist

### Source modifications

SourceMods

### Sandbox

Requested sandbox or tag

### Contact info

Contact information

### Any other relevant information

Enter relevant info

**Comment by fobryan3** [comment]
I can't provide any guidance on the NUOPC question.
I fully endorse the idea of having a more realistic and documented snow IC, but am curious why you advocate for a B-case rather than F-case to generate the forcing data for the I-case. That brings in drift in the atmos. lower boundary condition associated with ocean spinup itself. An alternative if you really want to go with B is to turn on salinity restoring in the ocean for this case to maintain Lab Sea convection.

**Comment by adamrher** [comment]
> but am curious why you advocate for a B-case rather than F-case to generate the forcing data for the I-case. That brings in drift in the atmos.

Good point. An F-case may be more appropriate. But the F-cases always tend to be a-bit warmer (usually about +1 or +2 W/m2 RESTOM), and so don't approximate the B-cases well. I suppose it's a good question to ask whether the drift in the B-case is worse than the mean bias in an F-case. All I'll say is that a little drift in the forcing streams seems to be OK. Here's an example of an offline CLM spin-up, using F-case for the forcing streams, where the forcing is 20 years. There is drift from the "snow free" first year, and the snow covered year 20, of about 0.25K globally.

**Comment by gustavo-marques** [comment]
@adamrher: you can save CPL history files by setting XML variables `HIST_N`  and `HIST_OPTION`.

**Comment by adamrher** [comment]
I just spoke to Mariana. We need to debug the cpl history output option in nuopc over the next week before we can do the snowpack spinup. I will keep everyone posted.
